### PR TITLE
Creating a Data Model 🎉 🚀 ✔️ 

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Creating a Data Model 🎉 🚀 ✔️ 

- Created a new app. Give it a meaningful name.
- Initialized your project with Git.
- Maked sure that your project has a Postgres database set up.
- Builded your project schema.

### CODE REVIEW

File: db/migrate/20221029174717_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```
Comment (Deepakdanger) [inline db/migrate/20221029174717_create_users.rb:5]: - [ ] `photo` column should be `string` and not `text`. Please fix this.

File: db/migrate/20221029174828_add_user_ref_to_posts.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```
Comment (Deepakdanger) [inline db/migrate/20221029174828_add_user_ref_to_posts.rb:5]: - [ ] For all the `posts`, `comments` and `likes` table, we will use user as `author` like I have shown below as per [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram.

Hint:
```
add_reference :posts, :author, foreign_key: { to_table: 'users' }
```

Use this hint for comments and likes as well.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Nizamuddin4493, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] I would recommend making the README professional including the project title, description, etc. You can follow the [design template](https://github.com/microverseinc/readme-template) given by the microverse.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Nizamuddin4493 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- Successfully implemented the change 👏 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**